.github/workflows/ci.yml

```diff
@@ -47,7 +47,9 @@ jobs:
         if: matrix.toolchain == 'stable' && matrix.os == 'ubuntu-latest'
 
       - run: cargo test --all-targets
-      - run: cargo test --features=symphonia-all --all-targets
+      - run: cargo test --all-targets --features=experimental
+      - run: cargo test --all-targets --features=symphonia-all
+      - run: cargo test --doc
```

Comment:
doc tests run be default with just cargo test. The only thing `--doc` does is only run doc tests

Reply:
Yes, `cargo test` runs doc tests, but `cargo test --all-targets` does not. 
```
 cargo test --help
Execute all unit and integration tests and build examples of a local package
...
Target Selection:
      ...
      --all-targets       Test all targets (does not include doctests)
      --doc               Test only this library's documentation
...
```

Then maybe just run `cargo test` without `--all-targets`?

Reply:
Updated.